src/Components/header/components/ChooseComponent.tsx

```diff
@@ -1,47 +1,87 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { AiFillCaretDown } from "react-icons/ai";
+import { useSearchParams } from "react-router-dom";
+import { QUERY_PARAMS } from "../../../Product/constants";
+import { HomePageTextsType, TypesTexts } from "../../../utils/interface";
```

Comment:
```suggestion
import { HomePageTextsType, TypesTexts } from "../../../utils/interface";
```

all interfaces should contain names as follows: IProduct, ISingleProduct

Reply:
remove word type if its an `interface`